core/src/com/unciv/logic/simulation/Simulation.kt

```diff
@@ -112,22 +113,66 @@ class Simulation(
         var outString = ""
         for (civ in civilizations) {
 
-            val wins = winRate[civ]!!.value * 100 / max(steps.size, 1)
-            if (wins == 0) continue
+            val numSteps = max(steps.size, 1)
+            val expWinRate = 1f / civilizations.size.toDouble()
```

Comment:
Please note that civilizations contains also *minor civs* which are not expected to win, expected win should be 1/(major civs)

Reply:
Ah, fixed. Thanks